app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -97,6 +98,10 @@ open class HomeActivity : AppCompatActivity() {
 
         setupToolbarAndNavigation()
 
+        if (intent.getBooleanExtra(EXTRA_FINISH_ONBOARDING, false)) {
```

Comment:
nit: it might be good to add a comment mentioning this flag is for automation, or performance work, because if someone finds this flag in isolation and doesn't see it used anywhere else in the code, they may think it's unused and remove it.

Reply:
Yup, that would have been a great idea! I can't believe I didn't think of doing that. Great catch!